PurchasesTests/SubscriberAttributes/BackendSubscriberAttributesTests.swift

```diff
@@ -84,7 +80,11 @@ class BackendSubscriberAttributesTests: XCTestCase {
             ]
         ]
 
-        expect(receivedParameters.requestBody as? [String: [String: NSObject]]).to(equal(expectedBody))
+        let body = receivedParameters.request.requestBody as? [String: [String: NSObject]]
+
+        // This is implicitly checking that the request was .post
+        expect(body) == expectedBody
```

Comment:
I think still might want to check for post? We want to make sure its not a put, patch, or a delete (since a delete is also technically allowed a body 🙈 )

Reply:
We don't do put or patch right now but just future proofing 🤷‍♂️

Reply:
That is a great point. We don't have anything other than get or post right now, but that could change.
Let me add that in #1353 since that changes further there.

Reply:
Done here: #1359